website/docs/developer/research/research-deep-glacier-migration.mdx

```diff
@@ -0,0 +1,242 @@
+      prefix = "tmp/"
+    }
+  }
+
+```
+
+:::tip
+To migrate to a different storage class if needed, change `storage_class` under transition block to the desired class in the terraform code above.
+:::
+
+3. Wait 48 hours for all objects to migrate to deep archive.
+4. Update the storage type in the ORCA Catalog to the value of DEEP ARCHIVE. Below are three approaches that can accomplish this.
+- (Recommended) Have a lifecycle rule applied to the glacier bucket that will trigger a lambda function for every `s3:LifecycleTransition` event in the bucket. This lambda will automatically update the ORCA catalog status table for storage class to DEEP ARCHIVE at the object level.
+- Manually run a temporary lambda function that updates the catalog status table for storage class to DEEP ARCHIVE after a set amount of time (48+ hours).
+- Manually update the table column in the database using SQL.
+5. Validate internal reconciliation for mismatches in storage class.
+6. If no mismatch, disable/delete the lifecycle rules.
+
+A detailed sequence diagram that goes into more depth on the steps and actions needed for manually migrating data to DEEP ARCHIVE is shown below. The sequence diagram uses the recommended steps and approaches called out above.
+
+<MyImage
+    imageSource={useBaseUrl('img/Deep-Archive-Migration-workflow-manual.svg')}
+    imageAlt="System Context"
+    zoomInPic={useBaseUrl('img/zoom-in.png')}
+    zoomOutPic={useBaseUrl('img/zoom-out.png')}
+    resetPic={useBaseUrl('img/zoom-pan-reset.png')}
+/>
+
+<br />
+<br />
+<br />
+
+The following is a container level diagram of the proposed architecture for the recommended manual method.
+
+<MyImage
+    imageSource={useBaseUrl('img/ORCA-Deep-Archive-Migration-Architecture-manual.svg')}
+    imageAlt="System Context"
+    zoomInPic={useBaseUrl('img/zoom-in.png')}
+    zoomOutPic={useBaseUrl('img/zoom-out.png')}
+    resetPic={useBaseUrl('img/zoom-pan-reset.png')}
+/>
+
+<br />
+<br />
+<br />
+
+:::note 
+The diagrams above are the initial thoughts for how a migration may be implemented. Changes to the workflow and architecture are likely as the migration is prototyped and validated.
+:::
+
+
+
+### Migrating specific ORCA holdings to deep archive.
+
+It is expected in certain instances, a DAAC operator will not be able to use the manual approach because of configurations, key path groupings, data selectivity or other factors. In order to be more selective about the data that is migrated to DEEP ARCHIVE, the following solution is proposed.
+
+Generally speaking, the migration of data would be initiated by an operator through the Cumulus Console and two steps would occur. First, the data to be migrated would be recovered from the current Flexible Storage (Glacier) holdings. Second, the recovered data would be copied back to the ORCA bucket in the DEEP GLACIER storage class. The steps below go into a general flow of how the events would occur and the user interact during migration. 
```

Comment:
First, the data to be migrated would be restored

Recovery refers to our internal Orca processes.

Reply:
changed